code/parachain/frame/dex-router/src/lib.rs

```diff
@@ -163,7 +163,8 @@ pub mod pallet {
 			asset_pair: CurrencyPair<T::AssetId>,
 			route: Option<BoundedVec<T::PoolId, T::MaxHopsInRoute>>,
 		) -> DispatchResult {
-			T::UpdateRouteOrigin::ensure_origin(origin)?;
+			T::UpdateRouteOrigin::ensure_origin(origin)?; // blas: why ignore output value?
```

Comment:
Since the UpdateRouteOrigin is RootOrCouncil, this is the standard way to make that generic.